### PR TITLE
[Tools] Enable PHPCS for CouchDB_Import_RadiologicalReview.php

### PR DESCRIPTION
**Pull request by laemtl**
Replaces #5998

### CODE REVIEW

**Review by sruthymathew123 — APPROVED** [review]

File: tools/CouchDB_Import_RadiologicalReview.php [hunk 1]
```diff
@@ -93,90 +124,112 @@ function __construct() {
...
-            WHEN r.SAS <> r.SAS2 THEN 'prim_second'
-            WHEN r.PVS <> r.PVS2 THEN 'prim_second'
-            ELSE 'false' END as Conflict_Any
-            FROM final_radiological_review frr
-            LEFT JOIN flag f ON (f.CommentID=frr.CommentID) 
-            LEFT JOIN session s ON (s.ID=f.SessionID) 
-            LEFT JOIN candidate c ON (c.CandID=s.CandID)
-            LEFT JOIN examiners eFinal ON (eFinal.ExaminerID=frr.Final_Examiner)
-            LEFT JOIN examiners eExtra ON (eExtra.ExaminerID=frr.Final_Examiner2)", array());
+            $query = <<<QUERY
```
Comment (driusan) [inline tools/CouchDB_Import_RadiologicalReview.php:147]: This should not be using heredocs, it's just a string.

**Review by driusan — CHANGES_REQUESTED** [review]

File: tools/CouchDB_Import_RadiologicalReview.php [hunk 2]
```diff
@@ -1,85 +1,116 @@
...
+ */
+class CouchDBRadiologicalReviewImporter
+{
     var $SQLDB; // reference to the database handler, store here instead
                 // of using Database::singleton in case it's a mock.
     var $CouchDB; // reference to the CouchDB database handler
 
+    // Ignoring PHPCS for now since this section is fairly readable and
+    // would require more refactoring to get right.
+    // phpcs:disable
```
Comment (driusan) [inline tools/CouchDB_Import_RadiologicalReview.php:36]: This should not be disabling phpcs

**Review by driusan — CHANGES_REQUESTED** [review]

**Comment by laemtl** [comment]
@driusan Ready

File: tools/CouchDB_Import_RadiologicalReview.php [hunk 3]
```diff
@@ -1,85 +1,134 @@
...
+        ],
+        'Conflict_Any'                   => [
             'Description' => 'Conflict between any reviews',
-            'Type' => "enum('No','Yes')"
-        )
-    );
+            'Type'        => "enum('No','Yes')",
+        ],
+    ];
+    // phpcs:enable
```
Comment (driusan) [inline tools/CouchDB_Import_RadiologicalReview.php:125]:
```suggestion
```

the disable was removed but the enable is still here

Comment (laemtl) [reply tools/CouchDB_Import_RadiologicalReview.php]: Fixed.

**Pull request merged**